frontend/src/pages/chat/Chat.tsx

```diff
@@ -516,7 +509,18 @@ const Chat = () => {
     }, [processMessages]);
 
     useEffect(() => {
-        if (AUTH_ENABLED !== undefined) getUserInfoList();
+        //prevent flash of auth message onload
+        const timer = setTimeout(() => {
+          if (AUTH_ENABLED === undefined) {
+            setShowAuthMessage(true);
+          }
+        }, 2000);
+        if (AUTH_ENABLED !== undefined) {
+          clearTimeout(timer);
+          if (!AUTH_ENABLED) return;
+          getUserInfoList();
+        }
+        return () => clearTimeout(timer);
```

Comment:
Won't this execute at some point during the 2000 wait on the `setTimeout`?

Reply:
The setTimeout is for checking if the value is undefined. The code above is in the check if the value is not undefined.